scripts/legislation/africa/namibia_english.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^58. Short title and commencement/,$p' | \
+    sed -n '/^PART I/,$p'
+}
+
+function prefix_article_numbers_with_article_literal {
+  sed -E 's/^([0-9]+\.)/Article \1/'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/'
+}
+
+function add_newlines_before_headers_and_articles {
+  local stdin="$(</dev/stdin)"
+
+  if [ "$#" -ne 1 ] ; then
+    echo_usage_error "$*" '<language>'
+    return 1
+  fi
+  local language="$1"
+
+  local article_regular_expression
+  article_regular_expression="$(get_article_regular_expression $language)"
+  local return_code="$?"
+  if [ "$return_code" -ne 0 ] ; then return "$return_code" ; fi
+
+  local regular_expression="([A-Z][A-Za-z' ]+) (${article_regular_expression})"
+
+  echo "$stdin" | sed -E "s/${regular_expression}/\n\n\1\n\2/g"
+}
+
+function remove_margin_headers {
+    sed -E 's/No. 3966 Government Gazette 27 December 2007//g' | \
+    sed -E 's/Act No. 7, 2007 ENVIRONMENTAL MANAGEMENT ACT, 2007//g' | \
+    sed -E 's/Government Gazette 27 December 2007 No. 3966 //g' 
+}
+
+function remove_leftover_article_literals {
+  sed -E 's/(in section Article \[44\])/in section 44/' | \
+  sed -E 's/Article \[([0-9]+)\]/\n(\1)/g'
+}
+
+function amend_errors_in_articles {
+    #remove incorrect bullet points
+    sed -E 's/ \[•\]//g' | \
```

Comment:
These two lines should not be indented to the same extent as the continuation lines.